inc/class-cachify.php

```diff
@@ -132,6 +132,30 @@ public function __construct() {
 			2
 		);
 
+		/* Add Cron for clearing the HDD Cache */
+		if ( self::METHOD_HDD == self::$options['use_apc'] ) {
```

Comment:
We would like to use more strict code, === instead of ==
please check and validate them, enhance the PR in this topic. Thx.

Reply:
Is included in "Make requested changes to checks"